lib/features/digital_guide_view/general_info/presentation/digital_guide_view.dart

```diff
@@ -53,9 +53,12 @@ class _DigitalGuideView extends ConsumerWidget {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final widgets1 = [
-      const SizedBox(height: DigitalGuideConfig.heightSmall),
-      MyCachedImage(
-        digitalGuideResponseExtended.imageUrl,
+      const SizedBox(height: DetailViewsConfig.heightImage),
+      SizedBox(
+        height: DetailViewsConfig.heightImage,
+        child: ZoomableOptimizedDirectusImage(
+          digitalGuideResponseExtended.imageUrl,
+        ),
```

Comment:
Is `SizedBox` declared twice needed here?

Reply:
Sorry there is a mistake in DetailViewsConfig.heightImage - I changed it, but it is needed as a visual gap